readme.md

```diff
@@ -71,7 +71,17 @@ Kindly ask you to raise the issue in case of any suggestions/bugs.
    ```bash
     java -jar docker-unittests.jar -f image-tests.yml -i openjdk:9.0.1-11
    ``` 
-   ![docker image tests results](./docs/image-tests-results.png)
+    ![docker image tests results](./docs/image-tests-results.png)
+
+#### General image test with output to xml file
+1. Add -o xml option to generate xml file
```

Comment:
Change text to 

1. Use `-o xml` option in order to receive the testing report in `xml` format